progenitor-impl/Cargo.toml

```diff
@@ -11,6 +11,7 @@ readme = "../README.md"
 heck = "0.4.1"
 getopts = "0.2"
 indexmap = "1.9"
+itertools = "0.10"
```

Comment:
is this used?

Reply:
Woud you mind if I add a `deny` rule?

Reply:
Addressed in `4323c11de94bc81c865868f8f10eea281bf62eb9`